cypress/integration/dashboard_spec.js

```diff
@@ -0,0 +1,40 @@
+describe('Dashboard User Flows', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3001');
+  });
+
+  it('Should display movie poster and ratings on individual movie cards on page load', () => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
```

Comment:
Nice intercept here. We discussed if a page load is technically a 'user flow'. I think at very least it is important that we did this in order to solidify the concept in our heads!